test_app/tests/test_forms.py

```diff
@@ -1,8 +1,15 @@
+import copy
+
+from django.conf import settings
 from django.test import TestCase
+from django.test.utils import override_settings
 
 from mapentity.forms import MapEntityForm
 from ..models import DummyModel
 
+mapentity_config = copy.deepcopy(settings.MAPENTITY_CONFIG)
+mapentity_config['MAX_CHARACTERS'] = 1200
```

Comment:
you should patch panetity settings as here:
https://github.com/makinacorpus/django-mapentity/blob/master/mapentity/tests/__init__.py#L305
https://github.com/makinacorpus/django-mapentity/blob/master/mapentity/tests/__init__.py#L430